<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DesignBox</class>
 <widget class="QGroupBox" name="DesignBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>892</width>
    <height>506</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox{
background-color:rgb(37, 40, 50);
color: white;
	border-radius: 15px;
}
QLabel{
color: white;

}
QLineEdit{
	border: 0px solid rgb(37, 39, 48);
	height: 30px;
	border-radius: 15px;
	color: white;
	padding-left: 20px;
	padding-right: 20px;
	background-color: rgb(27, 29, 35);
}
QLineEdit:hover{
	border: 2px solid rgb(64, 71, 88)
}
QLineEdit:focus{
	border: 3px solid rgb(91, 101, 124);
	color: white;
}
QLineEdit:disabled {
	background-color: #808080;
}
/* ToolButton */
QToolButton {
	border: 0px;
	width: 28px;
	height: 28px;
	background-color: transparent;
	color: rgb(255, 255, 255);
	border-radius: 15px;
}
QToolButton:hover {
	background-color: rgb(57, 65, 80);
	border: 2px solid rgb(61, 70, 86);
}
QToolButton:pressed {
	background-color: rgb(35, 40, 49);
	border: 2px solid rgb(43, 50, 61);
}
QPushButton{
	background-color: rgb(27, 29, 35);
}
QPushButton {
	border: 0px solid rgb(52, 59, 72);
	color: rgb(255, 255, 255);
	border-radius: 10px;
	margin-right:10px;
}
QPushButton:hover {
	background-color: rgb(57, 65, 80);
	border: 5px solid transparent;
}
QPushButton:pressed {
	background-color: rgb(35, 40, 49);
	border: 7px solid transparent;
}
QPushButton:disabled {
	background-color: rgb(102, 109, 122);
	color: rgb(168, 168, 168);
}

</string>
  </property>
  <property name="title">
   <string>Design</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="DesignPath_horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item>
      <widget class="QLabel" name="DesignPath_label">
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>11</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <property name="text">
        <string>Design Path</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="DesignPath_lineEdit">
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="BrowseDesignPath_button">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/icons/icons/folder_icon.png</normaloff>:/icons/icons/folder_icon.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="Configurations_horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item>
      <widget class="QPushButton" name="CompilationConfig_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>35</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Compilation configuration</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="LaunchingConfig_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>35</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Launching Configuration</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <property name="bottomMargin">
      <number>8</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="delete_pushButton">
       <property name="styleSheet">
        <string notr="true">background:none;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../IconsImages/delete.png</normaloff>../IconsImages/delete.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>35</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>66</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
